core/trino-main/src/test/java/io/trino/util/DynamicFiltersTestUtil.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.util;
+
+import io.trino.spi.predicate.SortedRangeSet;
+import io.trino.spi.type.Type;
+
+import java.util.StringJoiner;
+import java.util.stream.Collectors;
+import java.util.stream.LongStream;
+
+import static java.lang.String.format;
+
+public final class DynamicFiltersTestUtil
+{
+    private DynamicFiltersTestUtil() {}
+
+    public static String getSimplifiedDomainString(long low, long high, int rangeCount, Type type)
+    {
+        /* Expected simplified domain string is generated explicitly here instead of
+         * directly relying on output of Domain#toString so that we know when formatting of
+         * the output changes in some unexpected way.
```

Comment:
`so that we know when formatting of
         * the output changes in some unexpected way`?
        
Why it's important in this cae?

Reply:
updated